### ISSUE TITLE
JwtPayload TypeError: __init__() got an unexpected keyword argument 'aud'

### ISSUE THREAD

**Issue by EllieTheYeen** [MarshalX/atproto#181]
Now recently there has started to appear the following issue
```python
Traceback (most recent call last):
  File "/home/pi/bots/posters/crossposter/blueskypost.py", line 80, in <module>
    client._set_session(ses)
  File "/home/pi/.local/lib/python3.9/site-packages/atproto/xrpc_client/client/methods_mixin/session.py", line 68, in _set_session
    self._refresh_jwt_payload = get_jwt_payload(session.refresh_jwt)
  File "/home/pi/.local/lib/python3.9/site-packages/atproto/xrpc_client/client/auth.py", line 18, in get_jwt_payload
    return JwtPayload(**plain_payload)
TypeError: __init__() got an unexpected keyword argument 'aud'
Traceback (most recent call last):
  File "/home/pi/bots/posters/crossposter/blueskypost.py", line 80, in <module>
    client._set_session(ses)
  File "/home/pi/.local/lib/python3.9/site-packages/atproto/xrpc_client/client/methods_mixin/session.py", line 68, in _set_session
    self._refresh_jwt_payload = get_jwt_payload(session.refresh_jwt)
  File "/home/pi/.local/lib/python3.9/site-packages/atproto/xrpc_client/client/auth.py", line 18, in get_jwt_payload
    return JwtPayload(**plain_payload)
TypeError: __init__() got an unexpected keyword argument 'aud'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/pi/bots/posters/crossposter/blueskypost.py", line 88, in <module>
    client.login(username, password)
  File "/home/pi/.local/lib/python3.9/site-packages/atproto/xrpc_client/client/client.py", line 74, in login
  File "/home/pi/.local/lib/python3.9/site-packages/atproto/xrpc_client/client/client.py", line 40, in _get_and_set_session
    session = self._get_and_set_session(login, password)
    self._set_session(session)
  File "/home/pi/.local/lib/python3.9/site-packages/atproto/xrpc_client/client/methods_mixin/session.py", line 68, in _set_session
    self._refresh_jwt_payload = get_jwt_payload(session.refresh_jwt)
  File "/home/pi/.local/lib/python3.9/site-packages/atproto/xrpc_client/client/auth.py", line 18, in get_jwt_payload
    return JwtPayload(**plain_payload)
TypeError: __init__() got an unexpected keyword argument 'aud'
```
I think it has to do something with the JwtPayload

**Comment by fvillemin** [comment]
I confirm the issue, the auth doesn't work anymore :(

**Comment by MarshalX** [comment]
fix version will be released soon

**Comment by MarshalX** [comment]
fix version is `0.0.30` pls update using `pip install -U atproto`
